### PR TITLE
Add missing std namespace qualifiers

### PR DESCRIPTION
**Pull request by georgthegreat**
This is opencv_contrib part of [this PR](https://github.com/opencv/opencv/pull/24232) to opencv itself.

NB: we only build superres, optflow and ximgproc modules, so I can not be sure if all similar problems have been fixed.

### CODE REVIEW

File: modules/ximgproc/src/lsc.cpp [hunk 1]
```diff
@@ -821,7 +821,7 @@ inline void SuperpixelLSCImpl::PostEnforceLabelConnectivity( int threshold )
 
           (*Stmp).Neighbor.insert( (*Stmp).Neighbor.end(), (*S).Neighbor.begin(), (*S).Neighbor.end() );
 
-          sort( (*Stmp).Neighbor.begin(), (*Stmp).Neighbor.end() );
```
Comment (asmorkalov) [inline modules/ximgproc/src/lsc.cpp:824]: Stmp->Neighbor.begin(), Stmp->Neighbor.end()

Comment (georgthegreat) [reply modules/ximgproc/src/lsc.cpp:824]: I would like to avoid perfoming random refactorings in between.

This is almost automated change.

**Comment by georgthegreat** [comment]
@asmorkalov, can we merge this without the above refactoring?

**Review by asmorkalov — APPROVED** [review]
:+1:

**Pull request merged**